pkg/api/v1/descriptor/template.go

```diff
@@ -12,17 +12,17 @@ func init() {
 
 var templates = []definition.Descriptor{
 	{
-		Path:        "/pipelinetemplates",
-		Description: "Template API",
+		Path:        "/configtemplates",
+		Description: "Pipeline Config Template API",
 		Definitions: []definition.Definition{
 			{
 				Method:      definition.List,
 				Function:    handler.ListTemplates,
-				Description: "Get all cyclone built-in pipeline templates",
+				Description: "Get all cyclone built-in pipeline config templates",
```

Comment:
Also need to update description and some comments of function `ListTemplates`.

Reply:
have fixed